bin/oio-blob-rebuilder

```diff
@@ -35,58 +34,69 @@ before running this tool. This tool can also keep listening to a beanstalkd
 tube for broken chunks events.
 """
     parser = argparse.ArgumentParser(description=descr, parents=[log_parser])
+
+    # common
     parser.add_argument('namespace', help="Namespace")
-    parser.add_argument('--volume',
-                        help="Id of the volume to rebuild (IP:PORT)")
-    parser.add_argument('--dry-run', action='store_true',
-                        help="Display actions but do nothing")
-    parser.add_argument('--rdir-fetch-limit', type=int,
-                        help="Maximum of entries returned in "
-                             "each rdir response (100)")
-    parser.add_argument('--report-interval', type=int,
-                        help="Report interval in seconds (3600)")
-    parser.add_argument('--workers', type=int,
-                        help="Number of workers (1)")
-    parser.add_argument('--chunks-per-second', type=int,
-                        help="Max chunks per second per worker (30)")
-    parser.add_argument("--random-wait", type=int,
-                        help="Random wait (in microseconds)")
-    parser.add_argument('--allow-same-rawx', action='store_true',
-                        help="Allow rebuilding a chunk on the original rawx")
```

Comment:
I think we should keep the `--allow-same-rawx` option, and print a warning telling it is now enabled by default.